src/test/java/teammates/test/cases/ui/browsertests/AdminSessionsPageUiTest.java

```diff
@@ -63,13 +63,21 @@ private void testContent() {
         ______TS("content: hide filter");
         
         sessionsPage.clickDetailButton();
-        assertFalse(isTimeFramePanelVisible());
+        assertTrue(isTimeFramePanelInvisible());
         assertTrue(isSessionDataDisplayCorrect());
         
     }
     
     private boolean isTimeFramePanelVisible() {
-        return sessionsPage.isElementVisible(By.id("timeFramePanel"));
+        By timeFramePanel = By.id("timeFramePanel");
+        sessionsPage.waitForElementPresence(timeFramePanel);
+        return sessionsPage.isElementVisible(timeFramePanel) == true;
```

Comment:
Strictly speaking, this is not correct. The flow of this method is "wait for something to exist, then verify that the thing is visible". `waitForElementVisibility` should be the one that's used?